src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -520,11 +547,33 @@ std::vector<std::vector<int>> generate_stream_info(const int streams,
     return proc_type_table;
 }
 
-void get_num_streams(const int streams, const std::shared_ptr<ngraph::Function>& ngraphFunc, Config& config) {
+void get_num_streams(const int streams,
+                     const std::shared_ptr<ngraph::Function>& ngraphFunc,
+                     Config& config,
+                     const std::shared_ptr<InferenceEngine::IInferencePlugin>& plugin) {
     InferenceEngine::IStreamsExecutor::Config& executor_config = config.streamExecutorConfig;
     std::vector<std::vector<int>> proc_type_table = get_proc_type_table();
 
-    generate_stream_info(streams, ngraphFunc, config, proc_type_table);
+    ov::threading::IStreamsExecutor::Config streamsConfig;
+    streamsConfig._name = "CPUStreamsExecutor";
+    streamsConfig._streams = 1;
+    streamsConfig._threads = 1;
+    if (proc_type_table[0][MAIN_CORE_PROC] > 0) {
+        streamsConfig._streams_info_table.push_back({1, MAIN_CORE_PROC, 1, 0, 0});
+    }
+
+    if (streamsConfig._streams_info_table.empty()) {
+        generate_stream_info(streams, ngraphFunc, config, proc_type_table);
```

Comment:
I think it should not binding core when there is no Ecore.

Reply:
updated